docs/redoc/v0.9.0/openapi.json

```diff
@@ -0,0 +1,5093 @@
+    "contact": {
+      "email": "andrey@vasnetsov.com"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "master"
```

Comment:
```suggestion
    "version": "v0.9.0"
```

Reply:
Good catch, I completely missed that one :+1: 

I will rebase first to get your latest change on the transfer abort API.